pubgrade/modules/endpoints/builds.py

```diff
@@ -546,3 +577,87 @@ def delete_pod(name: str, namespace: str):
+# pubgrade.errors.exceptions.DeletePodError: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application. [foca.errors.exceptions]
+
+
+def trigger_signing_image(gh_action_path: str, cosign_private_key: str, cosign_password: str, dockerhub_token: str, gh_access_token: str, image_path: str):
+
+    username, password = base64.b64decode(dockerhub_token).decode('utf-8').split(":")
+    url = "https://api.github.com/repos/{}/dispatches".format("akash2237778/github-actions")
+    payload = json.dumps({
+        "event_type": "sign-image",
+        "client_payload": {
+            "cosign_key": cosign_private_key,
+            "docker_username": username,
+            "docker_password": password,
+            "cosign_password": cosign_password,
+            "image_path": image_path
+        }
+    })
+    headers = {
+    'Accept': 'application/vnd.github+json',
+    'Authorization': 'Bearer {}'.format("XXXXXXXXXXXXX"),
```

Comment:
This is not a good. TOKENs should be in variables, not in hardwired strings. Is this TOKEN stored at "gh_access_token" ?

Reply:
changed to variable